.gitignore

```diff
@@ -1,2 +1,2 @@
-
+vendor
```

Comment:
Please keep the vendor folder. We need it because we use a git sync plugin to update modules on the server and if we miss files it wouldn't work properly.